### DISCUSSION TITLE
VaporFile field does not work

### DISCUSSION THREAD

**Discussion by ibrahim-999** [laravel/nova-issues#4699] [Q&A]
Laravel Version: 8.82
Nova Version: 3.32.0
PHP Version: 7.4
Database Driver & Version: MySQL 8.0.17
Operating System and Version: linux mint
Browser type and version: chrome
Reproduction Repository: https://github.com/###/###

### Description:
i am using VaporFile instead of Nova file 

     ``` 
       VaporFile::make('File')
                ->updateRules('nullable')
                ->rules('required'),
      ```

### Detailed steps to reproduce the issue on a fresh Nova installation:

the create button freezes and nothing happen aftrer that  

![image](https://user-images.githubusercontent.com/66881245/184543130-e0bf16da-d107-4de9-bb77-4eecc467df1c.png)

**Comment by crynobone** [comment]
Please ensure all the requirements for uploading to Vapor S3 has been configured: https://docs.vapor.build/1.0/resources/storage.html#file-uploads

**Reply by ibrahim-999** [reply]
i have done all the configs that were needed to upload to vapor s3 and yet still my issue 

would you please help me @crynobone

**Reply by crynobone** [reply]
Based on your description above, following the documentation given above should already fix your problem. You haven't given us any additional information to work on. 

p/s: Don't ever tag anyone; Matt and Jeffrey don't work on Nova or Vapor.
